src/ClpIrV1Decoder.cpp

```diff
@@ -0,0 +1,224 @@
+#include "ClpIrV1Decoder.hpp"
+
+#include <emscripten/em_asm.h>
+#include <emscripten/val.h>
+
+#include <clp/ErrorCode.hpp>
+#include <clp/ffi/ir_stream/decoding_methods.hpp>
+#include <clp/ir/LogEvent.hpp>
+#include <clp/ir/LogEventDeserializer.hpp>
+#include <clp/ir/types.hpp>
+#include <clp/streaming_compression/zstd/Decompressor.hpp>
+#include <clp/TimestampPattern.hpp>
+#include <clp/TraceableException.hpp>
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include <span>
+#include <system_error>
+#include <utility>
+
+#include <spdlog/spdlog.h>
+
+#include "ClpJsException.hpp"
+#include "constants.hpp"
+
+using namespace std::literals::string_literals;
```

Comment:
Unused

Reply:
Seems to be needed by Line 122

Reply:
We should move it out side of the name space (line 24)

Reply:
Not sure if I got the the comment. Are you saying the `constants.hpp` header is unused? I also saw the warning with Clang-Tidy but that turns out to be a false one since we do need `cLogLevelNames` which is directly imported from the header.

Reply:
I think I get the comment now. Somehow the comment was posted on an outdated source. If I look into the latest source the line number you were pointing to makes sense.